### DISCUSSION TITLE
Feedback: Icon code samples

### DISCUSSION THREAD

**Discussion by dprothero** [twilio-labs/paste#2416] [Ideas] [answered]
The code sample that dynamically updates when you click an icon on the [icon list page](https://paste.twilio.design/components/icons) is great!

However, when you scroll down where the code sample is scrolled out of view, it's less helpful when clicking an icon far down on the list. You have to scroll all the way back to the top to get the code sample.

It'd be awesome if the code sample could be made sticky.

**Comment by nkrantz** [comment]
Hey @dprothero - just made a [ticket](https://issues.corp.twilio.com/browse/DSYS-3371) for this! Thanks again for the feedback and keep it coming!

**Comment by SiTaggart** [comment, marked as answer]
Hey @dprothero 

As if by magic, (or just @sandsng) this is now fixed on the docs site.  https://paste.twilio.design/components/icons

Thanks for your suggestion!
